main/platform.c

```diff
@@ -55,7 +55,9 @@ void target_reset(void) {
     esp_sleep_enable_timer_wakeup(20000);
     esp_deep_sleep_start();
     // just in case...
+#ifdef CONFIG_IDF_TARGET_ESP32
```

Comment:
was this function removed in newer IDF for C3?

Reply:
The function is only to ESP32 in newer esp-idf.

Reply:
Cool, could you do `#else` the regular reset?